src/julia_internal.h

```diff
@@ -376,6 +376,9 @@ static const int jl_gc_sizeclasses[] = {
     // pg = 2^14
     // sz = (div.(pg-8, rng).÷16)*16; hcat(sz, (pg-8).÷sz, pg .- (pg-8).÷sz.*sz)'
 
+#if 0 /* GC_SMALL_PAGE */
+    // TODO(netto): compute these tables
+#else
     // rng = 60:-4:32 (8 pools)
```

Comment:
@vtjnash, @gbaraldi: I'll likely need to recompute this table given the new page size.

Should `rng` remain the same, or it also needs to be updated (e.g. shrieked by a factor of 4) to reflect the new page size?